app/selenium_ui/conftest.py

```diff
@@ -246,19 +246,22 @@ def get_requests_by_url(requests, url_path):
     return filtered_requests
 
 
-def get_wait_browser_metrics(webdriver):
+def get_wait_browser_metrics(webdriver, expected_metrics):
     attempts = 20
```

Comment:
Please reduce the attempts to 10, to have a total possible sleep time 5 sec. And need to retest it with 5 secs and 10 attempts.

Reply:
Doesn't work properly with 10 attempts. Tested with 15 attempts, looks good to me.